File: src/main/java/io/github/abductcows/easyargs/annotations/CustomNonNullAPI.java
package io.github.abductcows.easyargs.annotations;


import javax.annotation.Nonnull;
import javax.annotation.meta.TypeQualifierDefault;
import java.lang.annotation.ElementType;

@TypeQualifierDefault({
        ElementType.FIELD,
        ElementType.METHOD,
        ElementType.PARAMETER,
        ElementType.TYPE_USE
})
@Nonnull
public @interface CustomNonNullAPI {}

File: src/main/kotlin/io/github/abductcows/easyargs/CustomNonNullAPI.kt
package io.github.abductcows.easyargs

import java.lang.annotation.ElementType
import javax.annotation.Nonnull
import javax.annotation.meta.TypeQualifierDefault


@Nonnull
@TypeQualifierDefault(ElementType.FIELD, ElementType.TYPE_USE, ElementType.METHOD, ElementType.PARAMETER)
annotation class CustomNonNullAPI